### ISSUE TITLE
Dead Space [NPUB30569]: Unplayable due to missing floor in chapter 2

### ISSUE THREAD

**Issue by Shoegzer** [RPCS3/rpcs3#9200]
At one point in chapter 2, you open a door to the "Imaging Diagnostics Wing" and you should be able to walk into the next room to progress, however the floor in the adjoininig room is missing and you simply fall to your death should you enter.  The immediate walls are missing too.  This prevents the game from being fully playable as it cannot be completed in this state.

I'm not convinced this isn't a bug in the game itself however, particularly since the game is known to have [many bugs](https://deadspace.fandom.com/wiki/Bugs_in_Dead_Space/Proposed) (though this particular bug is not mentioned), and the version I've tested is 01.00 so it is unpatched.   

RPCS3 with bug (floor and immediate walls not visible):

![screenshot](https://user-images.githubusercontent.com/12475371/97824920-3e870500-1c8b-11eb-9b70-94df534217ca.png)

Native system (floor visible):

![screenshot2](https://user-images.githubusercontent.com/12475371/97824944-4b0b5d80-1c8b-11eb-946f-41e54d311fe8.png)

Additionally [this](https://youtu.be/R30evY-u-Pg?t=634) video shows the door opening and the floor visible.

I've also suppled a save file below that loads right next to the door in question.  Would appreciate anyone else confirming that they experience the same issue with this save, or even confirming that they've been able to progress past this point in RPCS3 with a different save to narrow this down to a game vs. emulator bug.

Tested on Linux Mint 20, kernel 5.4 with RPCS3 at commit c4e89f8bb41f290e2c945a527c37601454307cb5 compiled using gcc 9.3.0.  Test system is an Intel i7-7700HK with Nvidia GTX 1070M using driver 450.80.  Tested against ppu/spu interpreter, accurate xfloat, opengl and vulkan, vsync, strict rendering mode, disable vertex cache, accurate rsx, and ppu llvm accurate vector NaNs.

[RPCS3.log.zip](https://github.com/RPCS3/rpcs3/files/5472538/RPCS3.log.zip)
[dead_space_gamesave.zip](https://github.com/RPCS3/rpcs3/files/5472467/dead_space_gamesave.zip)

**Comment by digitaldude555** [comment]
![Screenshot (155)](https://user-images.githubusercontent.com/26138653/97862286-bb5ac300-1d0d-11eb-84f4-3563e7878041.png)
Works fine for me. Log discord bot says "some game files are missing or corrupted, please re-dump and validate."

**Comment by Shoegzer** [comment]
Thanks @digitaldude555, that was actually quite helpful.  Taking a hint from your note about missing/corrupted game files I reinstalled the game and now it isn't giving me the issue so I'll close this.

By the way, what is a "log discord bot", and where did you get the information regarding missing/corrupted game files?  The log I posted above doesn't reflect missing files that I could tell.  Also, out of curiosity did you use my save specifically or did you progress from the beginning?
